### DISCUSSION TITLE
Back and Lay

### DISCUSSION THREAD

**Discussion by ktalovmorx** [cloudbet/docs#23] [Ideas]
When will both Back and Lay available?

**Comment by kgravenreuth** [comment]
Hi @ktalovmorx 

Thanks for asking about lay betting with Cloudbet API.

We already support Back betting, while we are actively looking into Lay betting too going forward. Whenever we have updates, we will definitely update here, so do subscribe to notifications here if you are interested in programmatic lay betting.

**Reply by placejam** [reply]
Hi @kgravenreuth 

Any update on this? Have we included lay as well?

**Reply by kgravenreuth** [reply]
Hi @placejam 

Thanks for raising this again. This is on our radar, while it hasn't been added yet. We will update you when an update is available.
